### PR TITLE
New Accounts Report

### PR DESCRIPTION
**Pull request by bbengfort**
### Scope of changes

This PR attaches a CSV file to the daily PLG report with all new accounts created in the past 24 hours.

Fixes SC-17174

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Describe how reviewers can test this change to be sure that it works correctly. Add a checklist if possible.

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [ ]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: pkg/quarterdeck/emails_test.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+				Invitations:   8,
+				Users:         1,
+			},
+		},
+	}
+
+	err := s.srv.SendDailyUsers(data)
+	require.NoError(err, "could not send daily users report")
+
+	// Check that there are two attachments
```
Comment (bbengfort) [inline pkg/quarterdeck/emails_test.go:63]: @pdeziel any recommendations on the best way to test this using the emails mock package?

File: pkg/quarterdeck/emails_test.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		Organizations:       87,
+		NewOrganizations:    0,
+		Projects:            87,
+		NewProjects:         1,
+	}
+
+	err := s.srv.SendDailyUsers(data)
+	require.NoError(err, "could not send daily users report")
+
+	// Check that there is only 1 attachment
```
Comment (bbengfort) [inline pkg/quarterdeck/emails_test.go:90]: @pdeziel same advice from above could be used here too!

File: pkg/utils/emails/builders.go [hunk 3]
```diff
@@ -123,22 +127,44 @@ type InviteData struct {
...
+	NewAccounts         []*NewAccountData `json:"-"`
+}
+
+// NewAccountData describes user accounts that were created in the last 24 hours. An
+// account is an instance of a user assigned to an organization. This includes the cases
+// where a user registers and creates an organization, where a user is invited to an
+// existing organization and creates an account, and where an existing user is invited
+// to a second organization. The organization data is from the perspective of the entire
+// organization not just the users' apikeys, projects, invitations, etc.
+type NewAccountData struct {
```
Comment (bbengfort) [inline pkg/utils/emails/builders.go:155]: @eschmier is this definition within your acceptance criteria?

**Review by pdeziel — APPROVED** [review]
Looks great!

**Pull request merged**